src/View/User/Includes/_inscription_form_course.html.twig

```diff
@@ -0,0 +1,56 @@
+{% block course_form_inscription %}
+    <div class="form col-xl-6 offset-xl-3">
+        <div class="form-title">
+            <h1>Formulaire d'inscription aux cours</h1>
+        </div>
+        <div>
+            <form class="form-course" method="POST">
+                <div class="mb-3 champ">
+                    <label for="firstname" class="form-label">Prénom de l'enfant :</label>
+                    <input type="text" class="form-control" id="firstname" name="firstname" placeholder="Delphine" value="" required>
+                </div>
+                <div class="mb-3 champ">
+                    <label for="lastname" class="form-label">Nom de l'enfant :</label>
+                    <input type="text" class="form-control" id="lastname" name="lastname" placeholder="Merlier" value="" required>
+                </div>
+                <div class="mb-3 champ">
+                    <label for="birthday" class="form-label">Date de naissance de l'enfant :</label>
+                    <input type="date" class="form-control" id="birthday" name="birthday" placeholder="" value="" required>
+                </div>
+                <div class="mb-3 champ">
+                    <p> Votre enfant est-il déjà monté sur un poney ? :</p>
+                    <select name="experience" class="form-label">
+                        <option>Oui</option>
+                        <option>Non</option>
+                    </select>
+                </div>
+                <div class="mb-3 champ">
+                    <p> A quel cours souhaitez vous inscrire votre enfant ? :</p>
+                    <select name="course" class="form-label">
+                        <option></option>
+                        <option></option>
+                        <option></option>
+                        <option></option>
```

Comment:
no placeholders while waiting for db values?

Reply:
Exactly for now i haven't linked this page with data bases, i'm waiting for their validation